docs/research.md

```diff
@@ -0,0 +1,33 @@
+# Research & Discovery
+
+## Existing similar tools and benefits to our project
+
+### 0xFableOrg/roll-op
+- Rollup customization with config file
+
+
+### upnodedev/opstack-compose
+- Dev ops solution (docker-compose + aws)
+- Upnode focuses on blockchain infrastructure (more value)
+- Has a block explorer
```

Comment:
I would add as a con:
- doesn't work out of the box
- Same regarding attachment to a version, the build process is hardcoded
- Same regarding deploying contracts from the same release used for building binaries

As pros:
- Does implement some deployment we can inspire ourselves on